src/test/java/milk/products/iot/MilkProductTest.java

```diff
@@ -21,9 +21,21 @@ void testToString() {
     }
 
     @Test
-    void testEqualsAndHash() {
+    void testEqualsAndHash0() {
```

Comment:
It's not customary to name tests this way.

Consider **renaming** it to 
```suggestion 
    void testEqualsAndHashMilkProduct() {
```
and so on.